katsdpdata/met_extractors.py

```diff
@@ -127,8 +127,20 @@ def _extract_metadata_from_katdata(self):
         int_secs = floor(self._katdata.start_time.secs)
         self.metadata['StartTime'] = time.strftime('%Y-%m-%dT%H:%M:%SZ', time.gmtime(int_secs))
         # targets are for archive string searches and radec/azel polute string searches
-        self.metadata['Targets'] = [t.name for t in self._katdata.catalogue.targets
-                                    if t.name not in ['None', 'Nothing', 'azel', 'radec']]
+        target_names = []
+        for t in self._katdata.catalogue.targets:
+            if t.name not in ['None', 'Nothing', 'azel', 'radec']:
```

Comment:
What about "blank" target names like
```
'Az: 57:17:44.8 El: 114:35:29.6'
'Ra: 3:49:10.99 Dec: 114:35:29.6'
```
You could chuck them with `startswith` if needed.

Reply:
Probably not... good plan.